components/OHPC_macros

```diff
@@ -154,3 +175,9 @@ Requires:      %{python_prefix}
     %endif \
 }
 
+%global ohpc_setup_optflags %{expand:\
+    export CFLAGS="%ohpc_optflags %ohpc_optflags_uarch"
+    export CXXFLAGS="%ohpc_optflags %ohpc_optflags_uarch"
+    export FFLAGS="%ohpc_optflags %ohpc_optflags_uarch"
+    export FCFLAGS="%ohpc_optflags %ohpc_optflags_uarch"
+}
```

Comment:
Would it make sense to also include the default RPM optflags for each architecture RPM defines?